test/system/src/test/java/io/pravega/test/system/ConsumptionBasedRetentionWithMultipleReaderGroupsTest.java

```diff
@@ -391,6 +409,119 @@ public void updateRetentionPolicyForCBRTest() throws Exception {
                 new StreamImpl(SCOPE_1, STREAM_2), 0L).join().values().stream().anyMatch(off -> off == 390));
     }
 
+    @Test
+    public void multipleControllerFailoverAndRestartCBRTest() throws Exception {
+        String scope = "testCBR2Scope" + RandomFactory.create().nextInt(Integer.MAX_VALUE);
+        String stream = "multiControllerStream" + RandomFactory.create().nextInt(Integer.MAX_VALUE);
+        String readerGroupName = "testmultiControllerReaderGroup" + RandomFactory.create().nextInt(Integer.MAX_VALUE);
```

Comment:
Nit: You can create the rand once and call nextInt 3 times.

Reply:
done